### PR TITLE
Add snap support steps into readme file

### PR DESCRIPTION
**Pull request by purveshpatel511**
Hey, I am adding snap installation guide into readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,17 @@ If you have Rust 1.36 or higher, you can install `hexyl` from source via `cargo`
 cargo install hexyl
 ```
 
+### Via snap package
+
+This package available on snapcraft with 5 different architechture `amd64`, `arm64`, `i386`, `ppc64el` and `s390x` with 4 different channel `stable`, `candidate`, `beta` and `edge`.
```
[the remark this answers is not in the document]
Comment (purveshpatel511) [reply README.md:106]: yeah, i would add that.

File: README.md [hunk 2]
```diff
@@ -101,6 +101,17 @@ If you have Rust 1.36 or higher, you can install `hexyl` from source via `cargo`
...
+
+You can install this package by running following commnad,
+```
+sudo snap install hexyl
+```
+
+This package is maintained by Github User [Purvesh Patel](https://github.com/purveshpatel511). You can find snap support repo [here](https://github.com/purveshpatel511/hexyl-snap). Feel free to open any snap related issues on [here](https://github.com/purveshpatel511/hexyl-snap/issues).
```
[the remark this answers is not in the document]
Comment (purveshpatel511) [reply README.md:113]: Thanks for pointing this out. I will move this information.

**Comment by purveshpatel511** [comment]
Hey, i modify Changelog and Readme file, as you suggested.

File: README.md [hunk 3]
```diff
@@ -103,14 +103,11 @@ cargo install hexyl
...
 
-This package available on snapcraft with 5 different architechture `amd64`, `arm64`, `i386`, `ppc64el` and `s390x` with 4 different channel `stable`, `candidate`, `beta` and `edge`.
-
-You can install this package by running following commnad,
 ```
 sudo snap install hexyl
 ```
 
-This package is maintained by Github User [Purvesh Patel](https://github.com/purveshpatel511). You can find snap support repo [here](https://github.com/purveshpatel511/hexyl-snap). Feel free to open any snap related issues on [here](https://github.com/purveshpatel511/hexyl-snap/issues).
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/hexyl)
```
Comment (sharkdp) [inline README.md:110]: I'm sorry, but I would prefer to have a plain text version here... for consistency with the other installation instructions.

Comment (purveshpatel511) [reply README.md:110]: sure man.

**Comment by purveshpatel511** [comment]
add plain text link in readme file.

**Pull request merged**